Source/Core/VideoCommon/BPStructs.cpp

```diff
@@ -214,9 +214,8 @@ static void BPWritten(const BPCmd& bp)
     srcRect.left = (int)bpmem.copyTexSrcXY.x;
     srcRect.top = (int)bpmem.copyTexSrcXY.y;
 
-    // Here Width+1 like Height, otherwise some textures are corrupted already since the native
-    // resolution.
-    // TODO: What's the behavior of out of bound access?
+    // The width/height of the region is off by 1 from what the game developer actually intended
+    // so add 1 to compensate
```

Comment:
This is just the definition of these structs. The WH structs contain one less than the width/height to make the edge case of 0 w/h impossible.